bin/runex

```diff
@@ -16,10 +16,24 @@
 # limitations under the License.
 
 main_class="$1"
-main_args="${*:2}"
+mvn_opts="-q"
+main_args=()
+
+shift
+while [[ $# -gt 0 ]] ; do
+  case $1 in
+    -d|--debug)
```

Comment:
Stealing arguments from the main class seems like it will be prone to errors... you never know if it was intended to be part of the main class or part of the script.

A better strategy would be to detect a DEBUG flag from the environment, as in:

```bash
DEBUG=true ./runex ...
```

or perhaps more familiar,

```bash
export DEBUG=true
./runex ...
```

Could also use something more explicit:

```bash
export RUNEX_MVN_ARG="-X"
./runex ...
```

Reply:
I like using an environment variable, I think that's the best solution.